cleansweep/helpers.py

```diff
@@ -150,6 +151,13 @@ def starts_with_vowel(input_text):
     return False
 
 @app.context_processor
+def google_image_url():
+    def get_google_photo_url(google_id):
+        google_profile_image_url = json.loads(urllib2.urlopen( "https://www.googleapis.com/plus/v1/people/"+google_id+"?fields=image&key="+app.config['GOOGLE_API_KEY']).read())['image']['url']
+        return google_profile_image_url
+    return dict(get_google_photo_url = get_google_photo_url)
+
```

Comment:
Please move the get_google_photo_url to the top-level and add it to the list of functions exposed in helpers function below. There is no need to write google_image_url and the decorator.